docs/designs/config.md

```diff
@@ -118,7 +118,10 @@ Configs is grouped by scenario:
 * `contribution`: to config whether to enable contribution, and where to config.
 * `contrtibutors`: to config which contributors to include or exclude.
 * `gitDependencies`: to config CRRs.
-Extension config is supported to share default config for all OPS repos, by `defaults: msdocs.yml`. Current config has higher priority than extension config when conflicts.
+Extension config is supported to share default config for all OPS repos, by `extend: msdocs.yml`. Current config has higher priority than extension config when conflicts. Extension config also supports array, whose latter item has higher priority.
+> [!NOTE]
+>
+> `dependencies` doesn't support `extend`, as we may import extending configs from dependent repos.
```

Comment:
having dependency repos in a shared location is a valid scenario, we may want to put localization config for all repos in a central repo,

Reply:
Yes, this should be supported. Changed the wording to make it clear...